packages/cozy-stack-client/src/ShortcutsCollection.js

```diff
@@ -14,16 +15,23 @@ class ShortcutsCollection extends DocumentCollection {
    * @param {string} attributes.name Filename
    * @param {string} attributes.url Shortcut's URL
    * @param {string} attributes.dir_id dir_id where to create the shortcut
+   * @throws {Error} - explaining reason why creation failed
    */
   async create(attributes) {
     if (!attributes.type) {
       attributes.type = SHORTCUTS_DOCTYPE
     }
-    if (!attributes.name || !attributes.url || !attributes.dir_id) {
+    if (!attributes.name.trim() || !attributes.url || !attributes.dir_id) {
```

Comment:
`attributes.name` may be undefined so you should check for nullity first before calling `.trim()` on it.